modules/33acrossBidAdapter.js

```diff
@@ -116,12 +115,11 @@ function interpretResponse(serverResponse) {
 }
 
 // Register one sync per bid since each ad unit may potenitally be linked to a uniqe guid
-function getUserSyncs(syncOptions) {
+function getUserSyncs(syncOptions, bidderRequests) {
```

Comment:
second argument here is `responses`. 
[This](https://github.com/prebid/Prebid.js/blob/master/src/adapters/bidderFactory.js#L342) calls adapter's getUserSync.

Since you need `bidderRequests` to do userSync. You can import `userSync` in your adapter and call `userSync.registerSync` from `interpretResponse` method. 

Here is the [hello_world](https://github.com/prebid/Prebid.js/blob/master/integrationExamples/gpt/hello_world.html) test page

Reply:
@jaiminpanchal27 Sorry, I am a confused. I was asked to introduce `bidderRequests` in `getUserSyncs` in https://github.com/prebid/Prebid.js/issues/1855. Please see @mkendall07  comments. Did I misunderstand what was being asked?

Reply:
@curlyblueeagle Sorry for this. You did exactly what was asked. We had some misunderstanding on the final outcome of the discussion.

Can you update with my suggestion.

Reply:
Np, np. I'll update as you suggested and let you know. Thanks for the clarification